filebeat/harvester/config.go

```diff
@@ -20,8 +21,10 @@ var (
 		BackoffFactor:   2,
 		MaxBackoff:      10 * time.Second,
 		CloseOlder:      1 * time.Hour,
-		ForceCloseFiles: false,
 		MaxBytes:        10 * (1 << 20), // 10MB
+		CloseRemoved:    false,
+		CloseRenamed:    false,
+		ForceCloseFiles: false,
```

Comment:
That would be great TBH. Lets do this as soon as support for this is in ucfg (not a priority).